openapi/anchor-platform/main.yaml

```diff
@@ -1,26 +1,241 @@
-openapi: 3.0.0
+openapi: 3.1.0
 info:
-  version: "2.0.0"
+  title: Anchor Platform
+  version: "2.8.4"
+  summary: The Anchor Platform is a set of tools and APIs that enable developers and businesses to build their own on and off-ramp services for the Stellar network. It provides a standardized interface, including the implementation of several Stellar Ecosystem Proposals (SEPs), to make it easy for businesses to integrate with Stellar-based wallets and exchanges.
   description: |
-    The Synchronous Callbacks API specification for the Stellar Anchor Platform project.
+    The Anchor Platform is a set of tools and APIs that enable developers and businesses to build their own on and off-ramp services for the Stellar network. It provides a standardized interface, including the implementation of several Stellar Ecosystem Proposals (SEPs), to make it easy for businesses to integrate with Stellar-based wallets and exchanges.
 
-    The Synchronous Callbacks API defines requests made by the Platform while it is processing a request from a client application. The
-    anchor's responses to these requests affect the Platform responses to the client application.
-  title: Synchronous Callbacks API
+    The platform also includes features for managing assets, transactions, and user accounts, and supports a variety of deployment options, including using Docker or Kubernetes. Overall, the Anchor Platform aims to simplify the process of building and managing a Stellar-based financial service, allowing businesses to focus on providing value to their customers.
+
+    Learn about integrating with the Anchor Platform in the [Admin Guide](https://developers.stellar.org/platforms/anchor-platform/admin-guide) or get API information in the [API Reference](https://developers.stellar.org/platforms/anchor-platform/api-reference).
+  termsOfService: https://stellar.org/terms-of-service
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
 tags:
```

Comment:
Everything below is copy of Platform API + Custody API right?

Reply:
correct

Reply:
Actually, Platform API + Custody API + Callbacks API

Reply:
Got it, any reason for merging them back together? I liked the previous one because it's easy to navigate, but don't have any strong preference. 
Also cc @philipliu for opinion on merging this yaml files together

Reply:
I would prefer if we could keep them separate

Reply:
The reasoning was that it's a bit of a "pre-emptive" move for the versioning setup. It would be much simpler to coordinate and manage the versions of a single specification file, rather than 3 or 4 different files.

That said, my mind has been moving in the same direction as you both over the last couple days. I think  there are a couple options to address it. I'm gonna dig a bit into the [versioning capabilities](https://docusaurus-openapi.tryingpan.dev/versioning) of the openapi plugin we're using, as well as this [experimental `join` command](https://redocly.com/docs/cli/commands/join) in the Redocly CLI. I think both could provide a reasonable option for what we're after (which is: A useful, maintainable specfile (or set of them) that cleanly and nicely generates the documentation pages for us).